packtools/sps/validation/tablewrap.py

```diff
@@ -163,23 +166,24 @@ def validate_table(self):
         Returns:
             The validation result in the expected format.
         """
-        if not self.data.get("table"):
-            return format_response(
-                title="table",
-                parent=self.data.get("parent"),
-                parent_id=self.data.get("parent_id"),
-                parent_article_type=self.data.get("parent_article_type"),
-                parent_lang=self.data.get("parent_lang"),
-                item="table-wrap",
-                sub_item="table",
-                validation_type="exist",
-                is_valid=False,
-                expected="table",
-                obtained=None,
-                advice="Identify the table",
-                data=self.data,
-                error_level=self.rules["table_error_level"],
-            )
+        table = self.data.get("table")
+        is_valid = bool(table)
+        return format_response(
+            title="table",
+            parent=self.data.get("parent"),
+            parent_id=self.data.get("parent_id"),
+            parent_article_type=self.data.get("parent_article_type"),
+            parent_lang=self.data.get("parent_lang"),
+            item="table-wrap",
+            sub_item="table",
+            validation_type="exist",
+            is_valid=is_valid,
+            expected="table",
+            obtained=table,
+            advice='Mark each table-wrap table encoding with <table> inside <table-wrap>. Consult SPS documentation for more detail.',
```

Comment:
@Rossi-Luciano A lógica deve ser:
`Mark [a coisa] com [a sub-tag] inside [tag]. Consult SPS documentation for more detail.'`
mas muda um pouco se é o caso de atributo.

ou seja, no lugar de:
`'Mark each table-wrap table encoding with <table> inside <table-wrap>. Consult SPS documentation for more detail.'`

faça:
`'Mark each **table** with **<table>** inside **<table-wrap>**. Consult SPS documentation for more detail.'`

Reply:
OK, @robertatakenaka